index.html

```diff
@@ -5,37 +5,117 @@
+        <ul class="sm">
+            <li><a href="facebook.com" class="fa fa-facebook smicon"></a></li>
+            <li><a href="twitter.com" class="fa fa-twitter smicon"></a></li>
+            <li><a href="#" class="smicon">English</a></li>
+            <li><a href="#" class="smicon">My page</a></li>
+            <li><a href="#" class="smicon">Logout</a></li>
+        </ul>
         <nav class="navbar">
+            <a href="index.html" class="logo"><img src="assets/1280px-Daum_Kakao_wordmark_-_2014.svg.png" alt="logo_img"></a>
             <ul class="menu-navbar">
                 <li class="menu-navbar__item">
-                    <a href="/webpages/about.html">about</a>
+                    <a href="pages/about.html" class="active">About</a>
+                </li>
+                <li class="menu-navbar__item">
+                    <a href="pages/program.html">Program</a>
+                </li>
+                <li class="menu-navbar__item">
+                    <a href="pages/join.html">Join</a>
+                </li>
+                <li class="menu-navbar__item">
+                    <a href="pages/sponsor.html">Sponsor</a>
+                </li>
+                <li class="menu-navbar__item">
+                    <a href="pages/news.html">News</a>
                 </li>
             </ul>
         </nav>
     </header>
     <main>
         <section class="home">
-
+            <h2 class="home-title">"Hello! Sharing world"</h2>
+            <h1 class="home-title__big">Creative commons global summit 2015</h1>
+            <p class="home-paragraph"> Lorem ipsum dolor sit, amet consectetur adipisicing elit. Sunt autem, delectus sit animi nam nihil, voluptatibus nobis natus atque quo qui blanditiis. Nobis ad autem nemo at reprehenderit ipsa blanditiis voluptas non nulla consequatur itaque voluptatibus.</p>
+            <h2 class="home-footer">2015.10.15(THU) ~ 16(FRI)</h2>
+            <h3 class="home-copywrite">&copy; National Museum of Korea, Art Center Nabi and more</h3>
         </section>
         <section class="main-program">
-
+            <h2 class="main-program__title">Main Program</h2>
+            <span class="main-program__line"></span>
+            <ul class="main-program__container">
+                <li class="main-program__list">
+                    <div class="list-head">
+                        <img src="/assets/lecture.svg" alt="list_image_one" class="list-head__img">
+                        <h3>Lecture</h3>
+                    </div>
+                    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Sunt, nisi esse.</p>
+                </li>
+                <li class="main-program__list">
+                    <div class="list-head">
+                        <img src="/assets/icon2.svg" alt="list_image_two" class="list-head__img">
+                        <h3>CC Exhibition</h3>
+                    </div>
+                    <p>Lorem ipsum dolor, sit amet consectetur adipisicing elit. </p>
```

Comment:
- [ ] Nice one fixing most of the required changes, but as the previous reviewer said [here (https://github.com/BenM4k/First_Capstone_Project/pull/1#discussion_r1171221722) on the fourth point that you should add an hover effect to the card, that is what the whiteness around one of those cards indicate, Kindly implement that 😉.